src/components/ListGroup.js

```diff
@@ -185,7 +186,7 @@ export default function ListGroup(props) {
           hasItems
             ? (
               <div
-                title={ `List contains ${items.length} item${items.length != 1 ? 's' : ''}` }
+                title={ translate(`List contains ${items.length} item${items.length != 1 ? 's' : ''}`) }
```

Comment:
Proposal: Feed number of items in as a context argument:

```suggestion
                title={ translate(`List contains {items} item${items.length != 1 ? 's' : ''}`, { items: items.length }) }
```

You'll otherwise need to provide an infinite list of translation strings.

Reply:
Thank you! I changed this line.